pkg/apis/core/v1beta1/constants/utils.go

```diff
@@ -34,3 +34,12 @@ func getDeprecatedAnnotation(annotations map[string]string, annotationKey, depre
 
 	return val, ok
 }
+
+// GetShootVPADeploymentNames returns the names of all VPA related deployments related to shoot clusters.
+func GetShootVPADeploymentNames() []string {
+	return []string{
+		DeploymentNameVPAAdmissionController,
+		DeploymentNameVPARecommender,
+		DeploymentNameVPAUpdater,
+	}
```

Comment:
Why `DeploymentNameVPAExporter` is missing?

Reply:
It's not deployed for the shoot VPA, only when the VPA is running in the `garden` namespace (on purpose), cc @wyb1